nf_n3iwf/Dockerfile

```diff
@@ -1,29 +0,0 @@
-FROM free5gc/n3iwf-base:latest AS builder
-FROM alpine:3.15
-
-LABEL description="Free5GC open source 5G Core Network" \
-    version="Stage 3"
-
-ENV F5GC_MODULE n3iwf
-ARG DEBUG_TOOLS
-
-# Install debug tools ~ 100MB (if DEBUG_TOOLS is set to true)
-RUN if [ "$DEBUG_TOOLS" = "true" ] ; then apk add -U vim strace net-tools curl netcat-openbsd ; fi
-
-# Install N3IWF dependencies
-RUN apk add -U iproute2
```

Comment:
This is specific to n3iwf and does not appear in the common Dockerfile that n3iwf is built from.